src/track/serato/markers.h

```diff
@@ -17,10 +17,10 @@ typedef std::shared_ptr<SeratoMarkersEntry> SeratoMarkersEntryPointer;
 
 class SeratoMarkersEntry {
   public:
-    enum class TypeId {
-        Unknown,
-        Cue,
-        Loop,
+    enum class TypeId : int {
+        Unknown = 0,
+        Cue = 1,
+        Loop = 3,
```

Comment:
Interesting, please add a source comment what happens with 2.

Reply:
I didn't encounter 2 yet. I'm pretty sure it is never used, as the markers tag has a fixed structure with exactly 5 cue entries (0 or 1) and 9 loop entries (3).

Reply:
Can you drop that info into the code?